README.md

```diff
@@ -131,54 +163,56 @@ for i in `ls -d ${INPUT_DIR}/sub-*`; do
 ```
 
-### Run `generate_renders.py`
+## Using `generate_renders.py`
 
 Generate 3D renders for every defaced image in the output directory.
 
   ```bash
   python dsst-defacing-pipeline/src/generate_renders.py -o ${OUTPUT_DIR}
   ```
 
-### Visual Inspection
+## Visual Inspection
 
 To visually inspect quality of defacing with [VisualQC](https://raamana.github.io/visualqc/readme.html), we'll need to:
 
-1. Open TurboVNC through an spersist session. More info [here](https://hpc.nih.gov/docs/nimh.html).
+1. Open TurboVNC through an spersist session. More info on [the NIH HPC docs](https://hpc.nih.gov/docs/nimh.html).
 2. Run the `vqcdeface` command from a command-line terminal within a TurboVNC instance
 
-  ```bash
-  sh ${OUTPUT_DIR}/QC_prep/defacing_qc_cmd
-  ```
+    ```bash
+    sh ${OUTPUT_DIR}/QC_prep/defacing_qc_cmd
+    ```
 
 ## Conceptual design
 
-1. Generate a ["primary" scans](#terminology) to [other scans'](#terminology) mapping file.
+1. Generate a ["primary" scans](#terminology) to [other scans](#terminology) mapping file.
 2. Deface primary scans
    with [@afni_refacer_run](https://afni.nimh.nih.gov/pub/dist/doc/htmldoc/tutorials/refacer/refacer_run.html) program
    developed by the AFNI Team.
 3. To deface remaining scans in the session, register them to the primary scan (using FSL `flirt` command) and then use
    the primary scan's defacemask to generate a defaced image (using `fslmaths` command).
 4. Visually inspect defaced scans with [VisualQC](https://raamana.github.io/visualqc) deface tool or any other preferred
    tool.
-5. Correct/fix defaced scans that failed visual inspection. See [here]() for more info on types of failures.
+5. Correct/fix defaced scans that failed visual inspection. See [here](FILLINTHEBLANK) for more info on types of failures.
 
 ![Defacing Pipeline flowchart](images/defacing_pipeline.png)
 
 ## Terminology
 
-While describing the process, we frequently use the following terms:
+While describing this process, we frequently use the following terms:
 
 - **Primary Scan:** The best quality T1w scan within a session. For programmatic selection, we assume that the most
   recently acquired T1w scan is of the best quality.
-- **Other/Secondary Scans:** All scans *except* the primary scan are grouped together and referred to as "other" or "
-  secondary" scans for a given session.
-- **Mapping File:** A JSON file that assigns maps a primary scan (or `primary_t1`) to all other scans within a session.
-  Please find an example file [here]().
-- **[VisualQC](https://raamana.github.io/visualqc):** A suite of QC tools developed by Pradeep Raamana (Assistant
+- **Other/Secondary Scans:** All scans *except* the primary scan are grouped together and referred to as "other" or
+  "secondary" scans for a given session.
+- **Mapping File:** A JSON file that assigns/maps a primary scan (or `primary_t1`) to all other scans within a session.
+  Please find an example file [here](FILLINTHEBLANK).
```

Comment:
@Arshitha Remember to FILLINTHEBLANK here.

Reply:
done!